src/output-json-dns.c

```diff
@@ -246,6 +249,7 @@ static struct {
    { "maila", LOG_MAILA },
    { "any", LOG_ANY },
    { "uri", LOG_URI }
+    // clang-format on
```

Comment:
Are we missing an update here because of which clang-format had changed the formatting? It ideally doesn't mess up untouched parts of code..

Reply:
YESS. I. missed the one line I added in last PR :(